README.md

```diff
@@ -72,3 +72,12 @@ following:
 
 This will subscribe to the events and update the DPUL records when they're
 created, updated, or deleted.
+
+## Replicate database and solr index from production to staging
+
+1. SSH to the staging machine as the system user
```

Comment:
`ssh pulsys@dpul-staging1 'sudo service nginx stop' && ssh deploy@dpul-staging1 'cd /opt/dpul/current && DATE=2021-023 bundle exec rake dpul:replicate:prod' && ssh pulsys@dpul-staging1 'sudo service nginx start'` ?

Maybe even a quick shell script that takes a DATE?